build/Modules/Umbraco.Build/Get-UmbracoBuildEnv.ps1

```diff
@@ -0,0 +1,181 @@
+#
+# Get-UmbracoBuildEnv
+# Gets the Umbraco build environment
+# Downloads tools if necessary
+#
+function Get-UmbracoBuildEnv
+{
+  # store tools in the module's directory
+  # and cache them for two days
+  $path = "$PSScriptRoot\temp"
+  $cache = 2
+  
+  if (-not (test-path $path))
+  {
+    mkdir $path > $null
+  }
+
+  # ensure we have NuGet
+  $nuget = "$path\nuget.exe"
+  $source = "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"
+  if ((test-path $nuget) -and ((ls $nuget).CreationTime -lt [DateTime]::Now.AddDays(-$cache)))
+  {
+    Remove-File $nuget
+  }
+  if (-not (test-path $nuget))
+  {
+    Write-Host "Download NuGet..."
+    $client = new-object Net.WebClient
```

Comment:
Instead of this can't we use `Invoke-WebRequest $source -OutFile $nuget` instead?